public/conduct.jade

```diff
@@ -0,0 +1,30 @@
+div.jsla-toplevel-container.jsla-content-page
```

Comment:
yeah your correct @ashwell we just do client side script redirects. In not sure of a way to setup alias for the pages since this would need to be something Harp provides.